File: level1/제일 작은 수 제거하기/Solution.java
import java.util.ArrayList;

class Solution {
    public int[] solution(int[] arr) {
        int min = arr[0];
        for (int a : arr) {
            if (a < min)
                min = a;
        }
        ArrayList<Integer> al = new ArrayList<>();
        if (arr.length == 1)
            al.add(-1);
        for (int element : arr) {

            if (element != min)
                al.add(element);
        }
        int[] answer = al.stream().mapToInt(Integer::intValue).toArray();
        return answer;
    }
}

File: level1/제일 작은 수 제거하기/Solution.py
def solution(arr):
    min = arr[0]
    if len(arr) == 1:
        return [-1]
    for i in arr:
        if (i <= min):
            min = i
    arr.remove(min)
    return arr

  # 배열의 최솟값 min(arr)
